### DISCUSSION TITLE
Controlled hours parking

### DISCUSSION THREAD

**Discussion by codeinabox** [streetcomplete/StreetComplete#5288] [Q&A] [answered]
I don't know if this is a term only used in the UK, I've noticed that public parking areas often have signs listing the controlled hours, during which you need to pay. For example, [here is an explanation from Southwark Council](https://www.southwark.gov.uk/parking/find-somewhere-to-park/pay-for-parking).

When I've come across these I've manually added tags, eg

```
fee=no
fee:conditional=yes @ (08:00-22:00)
```

Would this be a good contender for a new quest?

**Comment by arrival-spring** [comment, marked as answer]
It is in fact there already under "other answers", "depends on time and day"
